content/flux/v0.x/data-types/basic/time.md

```diff
@@ -2,10 +2,232 @@
+- [Operate on time](#operate-on-time)
+
+## Time syntax
+Time literals are represented by [RFC3339 timestamps](/influxdb/cloud/reference/glossary/#rfc3339-timestamp).
+
+```js
+YYYY-MM-DD
+YYYY-MM-DDT00:00:00Z
+YYYY-MM-DDT00:00:00.000Z
+```
+
+## Convert data types to time
+Use the [`time()` function](/flux/v0.x/stdlib/universe/time/) to convert
+the following [basic types](/flux/v0.x/data-types/basic/) to time:
+
+- **string**: must be formatted as an
+  [RFC3339 timestamp](/influxdb/cloud/reference/glossary/#rfc3339-timestamp).
+- **int**: parsed as a [Unix nanosecond timestamp](/influxdb/cloud/reference/glossary/#unix-timestamp)
+  and converted to a time value.
+- **uint**: parsed as a [Unix nanosecond timestamp](/influxdb/cloud/reference/glossary/#unix-timestamp)
+  and converted to a time value.
+
+```js
+time(v: "2021-01-01")
+// Returns 2021-01-01T00:00:00.000000000Z
+
+time(v: 1609459200000000000)
+// Returns 2021-01-01T00:00:00.000000000Z
+
+time(v: uint(v: 1609459200000000000))
+// Returns 2021-01-01T00:00:00.000000000Z
+```
+
+### Convert columns to time
+Flux lets you iterate over rows in a [stream of tables](/flux/v0.x/get-started/data-model/#stream-of-tables)
+and convert columns to time.
+
+**To convert the `_value` column to time**, use the [`toTime()` function](/flux/v0.x/stdlib/universe/totime/).
+
+{{% note %}}
+`toTime()` only operates on the `_value` column.
+{{% /note %}}
+
+```js
+data
+  |> toTime()
+```
+
+{{< flex >}}
+{{% flex-content %}}
+##### Given the following input data:
+| \_time                                            | \_value _<span style="opacity:.5">(int)</span>_ |
+| :------------------------------------------------ | ----------------------------------------------: |
+| {{% nowrap %}}2021-01-01T00:00:00Z{{% /nowrap %}} |                                     10000000000 |
+| {{% nowrap %}}2021-01-01T02:00:00Z{{% /nowrap %}} |                                     20000000000 |
+| {{% nowrap %}}2021-01-01T03:00:00Z{{% /nowrap %}} |                                     30000000000 |
+| {{% nowrap %}}2021-01-01T04:00:00Z{{% /nowrap %}} |                                     40000000000 |
+{{% /flex-content %}}
+
+{{% flex-content %}}
+##### The example above returns:
+| \_time                                            |  \_value _<span style="opacity:.5">(time)</span>_ |
+| :------------------------------------------------ | ------------------------------------------------: |
+| {{% nowrap %}}2021-01-01T00:00:00Z{{% /nowrap %}} | {{% nowrap %}}1970-01-01T00:00:10Z{{% /nowrap %}} |
+| {{% nowrap %}}2021-01-01T02:00:00Z{{% /nowrap %}} | {{% nowrap %}}1970-01-01T00:00:20Z{{% /nowrap %}} |
+| {{% nowrap %}}2021-01-01T03:00:00Z{{% /nowrap %}} | {{% nowrap %}}1970-01-01T00:00:30Z{{% /nowrap %}} |
+| {{% nowrap %}}2021-01-01T04:00:00Z{{% /nowrap %}} | {{% nowrap %}}1970-01-01T00:00:40Z{{% /nowrap %}} |
+{{% /flex-content %}}
+{{< /flex >}}
+
+**To convert any column to time**:
```

Comment:
Yes, this can be used for the `_value` column. `toTime()` is a wrapper function (alias) for this exact operation, but is hard-coded to only work with the `_value` column (the most common use case). Here's the function definition:

```js
toTime = (tables=<-) => tables
  |> map(fn: (r) => ({ r with _value: time(v: r._value) }))
```